testing/libmustache/APKBUILD

```diff
@@ -0,0 +1,52 @@
+# vim: smartindent noexpandtab tabstop=4 shiftwidth=4
+# Maintainer: Renoir Boulanger <hello@renoirboulanger.com>
+
+pkgname=libmustache
+_pkghash=c809212c2941f5aa8030ce791efa38a22f43af62
+pkgver=0.4.2
+#_urlver=v${pkgver}
+_urlver=${_pkghash}
+#_pkgver=${pkgver}
+_pkgver=${_pkghash}
+pkgrel=1
+pkgdesc="C++ implementation of Mustache intended mainly for use as a PHP extension"
+url="https://github.com/jbboehr/libmustache"
+arch="all"
+license="MIT"
+depends=""
+makedepends="autoconf automake libtool yaml-dev"
+subpackages=""
+_mustachespec_rev="b96be9fd4c6d6984828d93169fe7e86d8a8aec2f"
+_pkg_filename=${pkgname}-${_pkgver}
+source="${_pkg_filename}.tar.gz::https://github.com/jbboehr/${pkgname}/archive/${_urlver}.tar.gz
+        mustache-spec-${_mustachespec_rev}.tar.gz::https://github.com/jbboehr/mustache-spec/archive/${_mustachespec_rev}.tar.gz"
+builddir="${srcdir}/${_pkg_filename}"
```

Comment:
I don't think a separate variable (`_pkg_filename`) is needed for the sole instance of where it's used.